pyanaconda/modules/common/structures/storage.py

```diff
@@ -140,6 +141,18 @@ def children(self) -> List[Str]:
     def children(self, value):
         self._children = value
 
+    @property
+    def links(self) -> List[Str]:
+        """Children of the device.
+
+        :return: a list of device paths
```

Comment:
Why is it called "links" if it's "children_paths" ?

Reply:
I think it's because these are actually symlinks to the devices?

Reply:
Sorry I need to update the docstring, it's not children, it's symlinks indeed.